docs/README.md

```diff
@@ -1,10 +1,66 @@
 # Documentation Overview
 
+<!-- - [SRT API Documents](#srt-api-documents)
+- [Build Instructions](#build-instructions)
+- [Development Documents](#development-documents)
+- [Features](#features)
+- [Sample Applications](#sample-applications)
+- [Miscellaneous](#miscellaneous) -->
+
 ## SRT API Documents
 
-| Folder Name | File Name                                              | Description                                         | Refer as               |
-| :---------: | ------------------------------------------------------ | --------------------------------------------------- | ---------------------- |
-| API         | [API.md](API/API.md)                                   | Detailed description of the SRT C API               | SRT API                |
-| API         | [API-functions.md](API/API-functions.md)               | Reference document for SRT API functions            | SRT API Functions      |
-| API         | [API-socket-options.md](API/API-socket-options.md)     | Instructions and list of socket options for SRT API | SRT API Socket Options |
-| API         | [statistics.md](API/statistics.md)                     | How to use SRT socket and socket group statistics   | SRT Statistics         |
+|            Folder             | File Name                                          | Description                                          | Refer as                      |
```

Comment:
```suggestion
|            Folder             | File Name                                          | Description                                          | Refer as                      |
```
Not sure of the purpose of the "Refer as" column. Maybe change the column name to something more descriptive.

Reply:
In this column I put an official name of a document which should be used when referring to this document from another place, so that through all the documents we have pretty much the same format of doing this.

Reply:
Any suggestions to a better column name?

Reply:
Okay, I see now. I suggest changing "Refer as" to "Document Title", and maybe moving that column before Description. I would also put the link to the actual document in the Document Title instead of the file name, to emphasize how such links should appear elsewhere.